vignettes/C_Colony_events.Rmd

```diff
@@ -289,15 +283,15 @@ apiary <- buildUp(apiary)
 
 # Supersedure
 
-Supersedure is a replacement of a queen by her daughter without interference of
-the beekeeper. Supersedure is a natural way of requeening without swarming.
-There are many reasons for supersedure: poor physical condition of a queen, old
-age, diseases, depleted spermatheca, poorly bread queen, reduced pheromone
-output and many others (Hamdan, 2010).
+Supersedure is a replacement of a queen by her daughters without interference of
```

Comment:
It is only one daughter that replaces the queen

Reply:
Eventually, but that happens after infighting and if she successfully mates. So, quite a lot is happening there before all this is done. But see your point and will adapt.

Reply:
Done